src/test-apps/happy/test-templates/ServiceAccountManager.py

```diff
@@ -32,14 +34,32 @@
 import grpc
 from grpc.framework.interfaces.face.face import ExpirationError
 from httplib import HTTPSConnection
-from happy.Driver import Driver
```

Comment:
If we are removing this file from having dependencies on Happy, then it needs to be moved out of src/test-apps/happy/bin/... and then needs to be moved somewhere else.

However, unlike the other "bin/..." file which I do not suspect is used anywhere else in the unit and functional tests, this is a test template that undergirds other service-related tests.

By gutting it of its happy dependencies, you are likely breaking its ability to fulfill its original goal and role as a test template. Consequently, you will likely need to do some refactoring to create a common base of functionality on which this test template and the tool you are trying to effect can depend.

Reply:
I'm not sure this is quite right -- this file has been misplaced to begin with, and does not fulfill the definition of a test template.  Regrettably, this area of our tests needs quite a bit of attention, down to the naming (I'm not sure that any of the classes there really fill the role of templates)